cypress/component/DataSubmission/consent_group.spec.js

```diff
@@ -0,0 +1,105 @@
+import ConsentGroupForm from '../../../src/components/data_submission/ConsentGroupForm';
+import { cloneDeep } from 'lodash/fp';
+import { mount } from 'cypress/react';
+
+const props = {
+  idx: 0,
+  parentConsentGroup: {},
+  saveConsentGroup: () => {},
+  deleteConsentGroup: () => {},
+};
+
+let propCopy = {};
+
+beforeEach(() => {
+  propCopy = cloneDeep(props);
+});
+
+describe('Consent Group', function () {
+  it('Edits without saving', function () {
+    cy.spy(propCopy, 'saveConsentGroup');
+
+    mount(<ConsentGroupForm {...propCopy}/>);
+
+    cy.get('#0_consentGroupName').type('Hello!');
+    cy.get('#0_url').type('https://www.asdf.gov');
+
+    expect(propCopy.saveConsentGroup).to.not.be.called;
+  }),
+  it('Saves properly', function () {
+    cy.spy(propCopy, 'saveConsentGroup');
+
+    mount(<ConsentGroupForm {...propCopy}/>);
+
+    cy.get('#0_consentGroupName').type('Hello!');
+    cy.get('#0_url').type('https://www.asdf.gov');
+    cy.get('#0_primaryRadio_hmb').check();
+    cy.get('#0_col').check();
+    cy.get('#0_dataLocation_text').type('Not Determined{enter}');
+    cy.get('#0_saveConsentGroup').click().then(() => {
+      expect(propCopy.saveConsentGroup).to.be.calledWith({
+            "value": {
+                "consentGroupName": "Hello!",
+                "generalResearchUse": false,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 10 spaces but found 16.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9969451)
```suggestion
          "generalResearchUse": false,
```